src/scenes/LandingPage/LandingPage.tsx

```diff
@@ -22,41 +19,31 @@ function LandingPage() {
     {}
   );
 
-  const { push } = useRouter();
-
   const track = useTrack();
 
-  const handleEnterGallery = useCallback(() => {
-    // If the user is already authenticated, /auth will handle forwarding
-    // them directly to the feed
-    track('Landing page Sign In button click');
-    void push('/auth');
-  }, [push, track]);
-
-  const handleExploreClick = useCallback(() => {
-    track('Landing page Explore button click');
-    void push('/home');
-  }, [push, track]);
-
   return (
     <StyledLandingPage>
       <GalleryIntro />
       <Spacer height={24} />
       <StyledButtonContainer>
-        <SuppressedHrefWrapper href="/auth">
-          <Button text="Sign In" onClick={handleEnterGallery} dataTestId="sign-in-button" />
-        </SuppressedHrefWrapper>
+        <ButtonLink
+          href="/auth"
+          onClick={() => track('Landing page Sign In button click')}
+          dataTestId="sign-in-button"
+        >
+          Sign In
+        </ButtonLink>
         {isFeatureEnabled(FeatureFlag.FEED, query) && (
           <>
             <Spacer width={12} />
-            <SuppressedHrefWrapper href="/home">
-              <Button
-                text="Explore"
-                type="secondary"
-                onClick={handleExploreClick}
-                dataTestId="explore-button"
-              />
-            </SuppressedHrefWrapper>
+            <ButtonLink
+              href="/home"
+              onClick={() => track('Landing page Explore button click')}
+              dataTestId="explore-button"
+              variant="secondary"
+            >
+              Explore
+            </ButtonLink>
```

Comment:
Here's an example of how the upcoming refactor will look for true links. Much cleaner IMO!

Reply:
😍